code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -964,7 +964,18 @@
 	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY | CHEMICAL_GOAL_CHEMIST_BLOODSTREAM | CHEMICAL_GOAL_BOTANIST_HARVEST
 
 /datum/reagent/medicine/mannitol/on_mob_life(mob/living/carbon/C)
-	C.adjustOrganLoss(ORGAN_SLOT_BRAIN, -2*REM)
+	if(HAS_TRAIT(C, TRAIT_BRAIN_TUMOR)) // to brain tumor quirker
+		if(!overdosed) // don't merge this condition into the `if` above.
+			C.adjustOrganLoss(ORGAN_SLOT_BRAIN, -0.5*REM)
+			overdose_threshold = 35 // Brain tumor people have special overdose
```

Comment:
I want to put `on_mob_add()`, but the proc is currently bugged. I would fix it later.